crudsite/urls.py

```diff
@@ -21,7 +21,7 @@
 
 urlpatterns = [
     path('admin/', admin.site.urls),
-    path('api/',include("weights.urls")),
+    path('',include("weights.urls")),
```

Comment:
```suggestion
    path('api/',include("weights.urls")),
```

We want the API endpoints/URLs to still live under the /api/ directory. The weight tracker is formed of 2 constituent apps:

- The app itself that the consumer sees, which returns HTML pages
- An API that provides endpoints that serve data given a request, which returns serialised JSON data

As we're hosting both, we want to clearly delineate the two as they both serve different purposes.

Otherwise, we would have the following:

www.weighttracker.com/weight_tracker -HTML page
www.weighttracker.com/weights - API endpoint
www.weighttracker.com/persons - API endpoint
www.weighttracker.com/admin - HTML page

This can get confusing if a project starts growing to multiple pages, apps, or endpoints.

Reply:
To clarify, would we want the site to look something like:
www.weighttracker.com/weight_tracker -- HTML page
www.weighttracker.com/weight_tracker/weights --API endpoint?
www.weighttracker.com/admin -- HTML page
So instead of applying all the endpoints under API, i should rename it to weight_tracker?

Reply:
Or if I want them completely separate, have a .../weight_tracker url and a .../api/weights etc url?

Reply:
The latter:
 .../weight_tracker url and a .../api/weights etc url?